sdk/attestation/Azure.Security.Attestation/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Release History
+## 1.0.0-beta.1 (2020-12-08)
+First preview release. Functionally complete but cleanup is still required.
 ## 1.0.0-alpha.1 (2020-12-08)
```

Comment:
It should be `Unreleased` instead of a date to track all the notes for the upcoming unreleased version.  See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md for an example.  `Unreleased` is also effectively a marker for our build system to avoid accidentally publishing a the wrong package.

```suggestion
## 1.0.0-beta.1 (Unreleased)

## 1.0.0-alpha.1 (2020-12-08)
```

Reply:
Does this change if the alpha.1 was also unreleased?